docs/it/metadata_oidc_op.rst

```diff
@@ -46,42 +43,128 @@ Se un OP nei propri metadata non ha i claim **client_registration_types_supporte
-
-
-Ogni OP DEVE esporre all’interno dei propri metadati i seguenti claim obbligatori.
-
-.. list-table:: 
-   :widths: 20 20 40 20
-   :header-rows: 1
-
-   * - **Claim**
-     - **Tipo**
-     - **Descrizione**
-     - **Obbligatorio**
    * - **organization_name**
-     - String
      - Un nome leggibile che rappresenta l'organizzazione proprietaria dell'OP. 
      - |check-icon|
+   * - **issuer**
+     - URL HTTPS che identifica univocamente l'OP all'interno della Federazione.
+     - |check-icon|
+   * - **authorization_endpoint**
+     - URL HTTPS dell'Authorization Endpoint dell'OP, al quale gli RP sono rediretti per iniziare il flusso 
+       di autenticazione.
+     - |check-icon|
+   * - **token_endpoint**
+     - URL HTTPS del Token Endpoint dell'OP, che gli RP devono chiamare per scambiare il codice di autorizzazione 
+       per un ID Token, un Access Token ed eventualmente un Refresh Token.
+     - |check-icon|
+   * - **userinfo_endpoint**
+     - URL HTTPS dello UserInfo Endpoint dell'OP, che gli RP possono usare per ottenere informazioni consentite 
+       sull'utente autenticato.
+     - |check-icon|
+   * - **introspection_endpoint**
+     - URL HTTPS dell'Introspection Endpoint dell'OP, che può essere usato dagli RP per verificare la validità di 
+       un Access Token.
+     - |check-icon|
+   * - **revocation_endpoint**
+     - URL HTTPS del Revocation Endpoint dell'OP, che gli RP possono usare per revocare gli Access Token e/o i Refresh Token già rilasciati.
+     - |check-icon|
    * - **jwks**
-     - JSON
      - JSON Web Key Set :rfc:`7517#appendix-A.1`
      - |check-icon| in assenza del claim **signed_jwks_uri**. 
+   * - **jwks_uri**
+     - URL del JSON Web Key Set (JWKS) dell'OP, che contiene la/le chiavi pubbliche di cifratura/decifratura che l'RP deve usare.
+     - |check-icon|
    * - **signed_jwks_uri**
-     - String
-     - URL del JWT auto firmato e verificabile con la chiave pubblica di Federazione (JWK). 
-     - |check-icon| in assenza del claim **jwks**. 
+     - Un URL HTTP che punta a un JWT firmato che ha come payload il JWK Set dell'entità (vedere esempio sotto). 
+       Il JWT è firmato con una chiave inclusa nel JWK che l'entità ha pubblicato nel suo Entity Statement autofirmato.
+     - |uncheck-icon|
+   * - **response_types_supported**
+     - JSON Array che contiene una lista di *response_type* supportati dall'OP.
+     - |check-icon|
+   * - **response_modes_supported**
+     - JSON Array che contiene una lista dei valori dei *response_modes* supportati dall'OP.
+     - |check-icon|
+   * - **acr_values_supported**
+     - Array contenente i livelli SPID supportati dall'OP, rappresentati come URI. 
+       Deve contenere per SPID uno o più valori tra i seguenti:
+       
+       ``https://www.spid.gov.it/SpidL1``
+       ``https://www.spid.gov.it/SpidL2``
+       ``https://www.spid.gov.it/SpidL3``
+
+       Deve contenere per CIE uno o più valori tra i seguenti:
```

Comment:
CIE_L1 e soci sono deprecati, CIE adotta quelli SPID

@sp640919 attendo ai tuoi pdf, ti ribadisco di leggere la documentazione su google docs come condivisa, ritroviamo spesso deprecazioni

Reply:
in poche parole, rimuovi questi ACR CIE